src/chess/model/Pawn.java

```diff
@@ -84,10 +104,20 @@ public void setNewPiece(Piece newPiece) {
             if (isWhite()) {
                 if (position.substring(1, 2).equals("2")) {
                     if (board[indexNumber + 1][indexLetter].isEmpty() &&
-                            board[indexNumber + 2][indexLetter].isEmpty()) {
+                        board[indexNumber + 2][indexLetter].isEmpty()) {
                         possibilities.add(position.substring(0, 1) + 4);
                     }
                 }
+                if (position.substring(1, 2).equals("5")) {
+                    if (canCaptureEnPassant(position, board)) {
+                        if (!board[indexNumber][indexLetter + 1].isEmpty()){
+                            possibilities.add(ControlGame.numToLetter(indexLetter + 1) + (indexNumber + 2));
+                        }
```

Comment:
In order to remove the opponent's pawn, I want to set null after pawn move, but I don't know which part I should put this action.
board[indexNumber][indexLetter + 1].setPiece(null);

Reply:
Also, I'd like to add possibilities of En passant only after the opponent's pawn takes two steps forward, but I don't know how to make it happen.